.test-infra/jenkins/job_PreCommit_Whitespace.groovy

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PrecommitJobBuilder
+
+PrecommitJobBuilder builder = new PrecommitJobBuilder(
+    scope: this,
+    nameBase: 'Whitespace',
+    gradleTask: ':whitespacePreCommit',
+    triggerPathPatterns: [
+      '.*\\.md$',
+      '.*build\\.gradle$',
+    ]
```

Comment:
R: @damgad 
Hi Damian, sorry to bother you on this PR. Tyson told me you might be able to give some advice on this topic: does this regex look fine to you and could we test it before merging it? Thanks!

Reply:
Thanks, this is really helpful information!